Added the support for PyQt4 GUI

yashrsharma44 opened the pull request:
Added the support for GUI using PyQt4.
Support for #20

Commit:
Added the main.py with pyqt4

yashrsharma44 committed:
Updated README.md for GUI

Added the steps for the installation of GUI script

yashrsharma44 committed:
Merge pull request #1 from yashrsharma44/yashrsharma44-patch-1

Updated README.md for GUI

Parth-Vader commented on README.md:
```diff
@@ -59,6 +59,24 @@ Other than requiring Python3.x , you require the following libraries:<br>
+3. PyQt4 is not supported in Virtual Environment, so you need to setup PyQt manually in the `virtualenv`.
+4. Two dependencies need to be downloaded,PyQt4 and SIP.
+5. Download the binaries from the given website - 
+    PYQT4 - https://www.riverbankcomputing.com/software/pyqt/download  (Choose the Source Packages)
```
Add both of them in MarkDown Link form. [like this](github.com)

Parth-Vader commented on README.md:
```diff
@@ -59,6 +59,24 @@ Other than requiring Python3.x , you require the following libraries:<br>
+    SIP - https://www.riverbankcomputing.com/software/sip/download
+    
+6. In the folder, make a folder named build, and extract the PyQt4 and SIP, in the build folder.
+7. You can refer this youtube tutorial for rest part - https://www.youtube.com/watch?v=6NDVP5inphM
```
This is not how you write a README. You need to give full instructions - you can't just link a youtube video tutorial. Although you can write towards the end that the source was this video.

Parth-Vader commented:
Impressive work! But it's not working in my system.
I'm getting this error : `AttributeError: 'QString' object has no attribute 'startswith'`.

yashrsharma44 commented:
Can you state if you are using `virtualenv` or not?
Also try using python3.
IMHO, try to crosscheck with the steps that I have stated in the README.md

Parth-Vader commented:
@yashrsharma44 My bad, I accidentally used an old virtualenv with Python 2.7.
Great work :smile: 
Should this be added to the `master` branch or another branch `gui`? If to the master, I would like both the `graph.py` file to exist and this too. Also I'm requesting some changes.

Parth-Vader commented on main.py:
```diff
@@ -0,0 +1,142 @@
+from PyQt4 import QtGui
```
After this, another commit was included (for Windows functionality) - [this.](https://github.com/Parth-Vader/FB-Spider/commit/d9a373712be1a0f492c53a153c59e9ac83b886d8)
Make the suitable changes here too.

Parth-Vader commented on README.md:
```diff
@@ -59,6 +59,24 @@ Other than requiring Python3.x , you require the following libraries:<br>
    A file called 'Table.html' would be made in the folder itself.
    You can open it and see the table.
    
```
If you want these to be merged in `master`, you also need to add the latest `graph.py` file.
Also, changes in the README.md should be made clarifying these 2 paths to run the program,

Parth-Vader reviewed (changes requested):
Added some minor comments about adding `graph.py`, making suitable changes in README and adding some lines in `main.py`.

yashrsharma44 commented:
Thanks for your views. I will start working in it, in order to make sure it is merged in the master branch.
Sorry for not completing the README.md. I will make sure to complete them.

yashrsharma44 committed:
Added all the steps for running the GUI

Added all the steps, and created two halves one for CLI version and one for GUI version.

yashrsharma44 committed:
Corrected typo

yashrsharma44 committed:
Added the changes for the windows problem

yashrsharma44 committed:
Added graph.py

yashrsharma44 committed:
Added the PyQt and SIP in the requirement list

yashrsharma44 commented:
Added the changes.

Parth-Vader commented:
This was surely a great PR @yashrsharma44!
I would also want you to add the GIFs for GUI version, and maybe you could also come up with an icon for FB-Spider :wink:
